src/index.html

```diff
@@ -5,8 +5,325 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a href="#home">
+            <img src="./images/logo.png" alt="Moyo logo">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#apple">APPLE</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#samsung">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#smartphones">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a 
+                 class="nav__link" 
+                 href="#"
+                 data-qa="nav-hover"
+              >
+                Laptops & computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#gadgets">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#tablets">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#photo">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#video">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="catalog">
+      <div class="catalog__container">
+        <ul class="catalog_list">
+          <li class="catalog_item">
```

Comment:
ul and li are redundant here
ul usually used for displaying list of navigation, or smth else
it's not the list of cards, it's catalog
so let's remove ul and li elements, and leave just cards

Reply:
ok